src/components/Toast/Toast.st.css

```diff
@@ -0,0 +1,60 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color;
+}
+
+:vars {
+    successColor: #57C487;
+    errorColor: #F64D43;
+    statusColor: #212121;
+}
+
+.root {
+    -st-states: rtl, mobile, skin(enum(success, error, status));
+    display: inline-flex;
+    color: #FFFFFF;
+    border-radius: 2px;
+    font-size: 14px;
+    font-family: "HelveticaNeueW01-55Roma";
```

Comment:
You also need to provide fallbacks, in `wix-style-react` these are the fallbacks:
```
"HelveticaNeueW01-55Roma", "HelveticaNeueW02-55Roma", "HelveticaNeueW10-55Roma", "Helvetica Neue", "Helvetica", "Arial", "メイリオ, meiryo", "ヒラギノ角ゴ pro w3", "hiragino kaku gothic pro", "sans-serif"
```